events.html

```diff
@@ -9,12 +9,11 @@ <h1 class="h2 text-center mb-4 ">Events</h1>
 <div class="container events">
   <div class="row justify-content-center">
     <div class="col-md-8">
-      <p>Sustain started a single event: the San Francisco Summit in 2017. It then went on to London in 2018, and Brussels in 2020. As it has aged, it's become more and more clear that Sustain is more than just an event. Instead, it's a movement.</p>
+      <p>Sustain started a single event: the San Francisco Summit in 2017. It then went on to London in 2018, and Brussels in 2020. As it has aged, it's become more and more apparent that Sustain is more than just an event. Instead, it's a movement.</p>
 
-      <p>In 2020, we saw the first satellite events using the Sustain brand: one in Lagos, and one in Colorado. Looking forward, we're hoping to find new ways to host Sustain events aroudn the world, virtually and in person, in order to keep conversations around sustainability going.</p>
+      <p>In 2020, we saw the first satellite events using the Sustain brand: one in Lagos, and one in Colorado. Looking forward, we're hoping to find new ways to host Sustain events around the world, virtually and in person, to keep conversations around sustainability going.</p>
 
-      <p>All of our work and organizing is open source. If you're interested in how Sustain has hosted events, want to host your own event, or want to use the Sustain brand as part of your event, get in touch and let us know.</p>
-    </div>
+      <p>All of our work and organizing is open source. If you're interested in how Sustain has hosted events, want to host your event, or want to use the Sustain brand as part of your event, get in touch and let us know.</p>    </div>
```

Comment:
Not sure that removing 'own' helps, and this puts the div on the same line. Seems unjustifiable to me?

Reply:
Addressed in commit 9e922ad0e719beb01c8d9c16e97a4deaa9d5e863.